plone/app/upgrade/v61/profiles/uninstall-discussion/catalog.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<object name="portal_catalog">
+
+  <index meta_type="FieldIndex"
+         name="total_comments"
+         remove="true"
+  />
+  <index meta_type="KeywordIndex"
+         name="commentators"
+         remove="true"
+  />
+  <column remove="true"
+          value="total_comments"
+  />
```

Comment:
Just to be extra sure: deleting indexes or columns from the catalog does not cause a full reindex of the content, right? 🤔

Reply:
Correct.